frontend/src/resources/pod.ts

```diff
@@ -34,7 +34,7 @@ export interface PodList extends IResource {
   items: Pod[]
 }
 
-export function listHivePods(namespace: string, labels?: string[]) {
+export function listPods(namespace: string, labels?: string[]) {
```

Comment:
This isn't a necessary change, but when using this function myself it seemed more appropriate to state exactly what the function was doing in it's name, not how it was being implemented elsewhere.